R/net.utils.R

```diff
@@ -981,35 +976,38 @@ get_degree <- function(x) {
 #' @export
 #'
 #' @examples
+#' set.seed(12345)
 #' param <- param.icm(inf.prob = 0.2, act.rate = 0.25)
 #' init <- init.icm(s.num = 500, i.num = 1)
 #' control <- control.icm(type = "SI", nsteps = 200, nsims = 1)
 #' mod1 <- icm(param, init, control)
-#' df <- as.data.frame(mod1)
+#' df <- as.data.frame(mod1, out = "vals")
 #' print(df)
 #' plot(mod1)
+#' mod1$control$nsteps
 #'
 #' mod2 <- truncate_sim(mod1, at = 150)
 #' df2 <- as.data.frame(mod2)
 #' print(df2)
 #' plot(mod2)
+#' mod2$control$nsteps
 #'
 truncate_sim <- function(x, at) {
-
+  
   if (class(x) != "icm" && class(x) != "netsim") {
     stop("x must be either an object of class icm or class netsim",
          call. = FALSE)
   }
-
+  
   rows <- at:(x$control$nsteps)
-
+  
   # epi
   x$epi <- lapply(x$epi, function(r) r[rows, ])
-
+  
   # control settings
   x$control$start <- 1
   x$control$nsteps <- max(seq_along(rows))
-
+  
   return(x)
```

Comment:
All these additions of whitespace need to be reverted back. Might be a tab option you have set in Rstudio that generated all of them.